src/main/webapp/admin_dashboard_script.js

```diff
@@ -1,8 +1,8 @@
 var map;
 var styledMapType;
-var map_key = keys.MAP_KEY;
+var mapKey = config.MAP_KEY;
```

Comment:
Note that the variable name in the README is `MAPS_KEY`, not `MAP_KEY`.

Reply:
Is there a commit to push that renames this variable to `MAPS_KEY`, or updates the README to `MAP_KEY`?